PopUpWindow.js

```diff
@@ -0,0 +1,194 @@
+  inf.classList.add('title');
+  description.innerHTML = projects[key].description.substring(0, Math.max(130,
+    Math.floor(0.4 * projects[key].description.length)));
+  description.classList.add('project_description');
+
+  projects[key].technolgies.forEach((element) => {
+    techns.innerHTML = `${techns.innerHTML}<li>${element} </li>`;
+  });
+  techns.classList.add('skills');
+  techns.id = 'skills_0';
+  btnLoad.innerHTML = 'See Project';
+  btnLoad.href = '#';
+  projectSection.appendChild(general);
+  projectSection.id = key;
+
+  return projectSection;
+}
+
+Object.keys(projects).forEach((key) => {
+  myWork.appendChild(createMobileProjectCard(key));
+});
+
+const popUpBtn = document.querySelectorAll('.external_link');
+const btnSource = document.createElement('a');
+const btnLive = document.createElement('a');
+const externalButton = document.createElement('div');
+const wrapper = document.createElement('div');
+const home = document.querySelector('.starting-page');
+const cancelBtn = document.createElement('img');
+const imgWrapper = document.createElement('div');
+const header = document.createElement('div');
+const title = document.createElement('h3');
+const infos = document.createElement('p');
+const image = document.createElement('img');
+const description = document.createElement('p');
+const skills = document.createElement('ul');
+const fullwindow = document.createElement('div');
+
+imgWrapper.classList.add('pop-header-div');
+cancelBtn.src = 'assets/component.svg';
+imgWrapper.appendChild(cancelBtn);
+
+description.classList.add('project_description');
+
+infos.classList.add('inf');
+
+const bottom = document.createElement('div');
+const mobilePart = document.createElement('div');
+
+title.classList.add('pop-header-h3');
+header.appendChild(title);
+title.style.marginLeft = '32px';
+header.appendChild(imgWrapper);
+header.classList.add('pop-header');
+btnSource.innerHTML = '<img src="assets/liveView.png" alt="Live version"> ';
+btnLive.innerHTML = '<img src="assets/crossmark.png" alt="Live version"> ';
+externalButton.appendChild(btnLive);
+externalButton.appendChild(btnSource);
+
+bottom.appendChild(skills);
+bottom.appendChild(externalButton);
+
+mobilePart.appendChild(description);
+mobilePart.appendChild(bottom);
+mobilePart.classList.add('mobile-part-class');
+mobilePart.querySelector('p').classList.add('pop-header-description');
+infos.style.marginLeft = '32px';
+image.classList.add('project_image');
+image.classList.add('pop-header-img');
+
+wrapper.appendChild(header);
+wrapper.appendChild(infos);
+wrapper.appendChild(image);
+wrapper.appendChild(mobilePart);
+
+/* wrapper.style.zIndex = '999';
+wrapper.style.position = 'absolute';
+wrapper.style.top = '0'; rd */
+fullwindow.style.zIndex = '999';
+fullwindow.style.position = 'absolute';
+fullwindow.style.top = '0';
+fullwindow.style.width = '100%';
```

Comment:
- [ ] As asked by the previous reviewer and according to the design when popup is open please make the background blur

Reply:
hi @GzimAsani ,
thank you for mentioning the issue. can you please help to solve this issue?

Reply:
- Check this [Article](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter) about the `backdrop-filter` which may help you